vmc/resource_vmc_cluster_test.go

```diff
@@ -0,0 +1,117 @@
+/* Copyright 2020 VMware, Inc.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package vmc
+
+import (
+	"fmt"
+	"github.com/vmware/vsphere-automation-sdk-go/services/vmc/orgs"
```

Comment:
can the imports be rearranged. I thought that the style check does it automatically.

Reply:
In IntelliJ I have to run goimports tool for that. I will fix it

Reply:
Done